### PR TITLE
update package clipboard to @react-native-clipboard/clipboard

### PR DESCRIPTION
**Pull request by edritech93**
change package clipboard to @react-native-clipboard/clipboard
because can't run on react-native 0.73.x

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -916,4 +916,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ec00682c7062a323dff24a3c3643ca0bbb420d01
 
-COCOAPODS: 1.14.3
```
Comment (saghul) [inline ios/Podfile.lock:919]: Please don't change the CocoaPods version.

File: ios/app/app.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1031,7 +1031,8 @@
 				OTHER_CPLUSPLUSFLAGS = "$(inherited)";
 				OTHER_LDFLAGS = (
 					"$(inherited)",
-					" ",
```
Comment (saghul) [inline ios/app/app.xcodeproj/project.pbxproj:1034]: Why is this change necessary?

File: package.json [hunk 3]
```diff
@@ -1,213 +1,212 @@
...
-    "validate": "npm ls",
-    "tsc-test:web": "tsc --project tsconfig.web.json --listFilesOnly | grep -v node_modules | grep native",
-    "tsc-test:native": "tsc --project tsconfig.native.json --listFilesOnly | grep -v node_modules | grep web",
-    "start": "make dev"
-  },
-  "resolutions": {
-    "@types/react": "17.0.14",
-    "@types/react-dom": "17.0.14"
-  }
+    "name": "jitsi-meet",
```
Comment (saghul) [inline package.json:2]: Don't reformat the file please. Keep your changes to a minimum.

File: patches/@giphy+js-analytics+4.2.0.patch [hunk 4]
```diff
@@ -1,14 +0,0 @@
-diff --git a/node_modules/@giphy/js-analytics/dist/send-pingback.js b/node_modules/@giphy/js-analytics/dist/send-pingback.js
-index 989f0ff..149e77c 100644
---- a/node_modules/@giphy/js-analytics/dist/send-pingback.js
-+++ b/node_modules/@giphy/js-analytics/dist/send-pingback.js
```
Comment (saghul) [inline patches/@giphy+js-analytics+4.2.0.patch:4]: Do not remove our patches please.

**Review by saghul — CHANGES_REQUESTED** [review]

File: ios/Podfile.lock (hunk 1 above)
Comment (edritech93) [reply ios/Podfile.lock:919]: hi, it's just from template

File: ios/app/app.xcodeproj/project.pbxproj (hunk 2 above)
Comment (edritech93) [reply ios/app/app.xcodeproj/project.pbxproj:1034]: hi, it's just from template

File: patches/@giphy+js-analytics+4.2.0.patch (hunk 4 above)
Comment (edritech93) [reply patches/@giphy+js-analytics+4.2.0.patch:4]: oh sorry, i will revise

File: ios/app/app.xcodeproj/project.pbxproj (hunk 2 above)
Comment (saghul) [reply ios/app/app.xcodeproj/project.pbxproj:1034]: What Xcode version did you use?

File: package.json (hunk 3 above)
Comment (saghul) [reply package.json:2]: It doesn't matter where it comes from, we can't accept PRs that reformat files just because.

**Comment by danilvalov** [comment]
@edritech93 Any updates?
Your PR is really needed! Could you fix remaining fixes?

**Comment by danilvalov** [comment]
@edritech93 Sorry, but I took it upon myself to [re-create your pull request](https://github.com/jitsi/jitsi-meet/pull/14844) because I really need this to be fixed as quickly as possible, and you have not been active on this pull request for 3 months. I hope you don't mind.

I have included your original authorship there

**Comment by saghul** [comment]
Superseded by https://github.com/jitsi/jitsi-meet/pull/14844
